src/core/config.js

```diff
@@ -58,7 +58,25 @@ const config = {
 	callbacks: {},
 
 	// The storage module to use for reordering tests
-	storage: localSessionStorage
+	storage: localSessionStorage,
+
+	// Get the testIds for all tests, value is read only
+	get testIds() {
+		var i, j, ml, tl, testIds = [], testNames, testName;
+
+		// Loop through all modules and tests and generate testIds
+		for ( i = 0, ml = config.modules.length; i < ml; i++ ) {
+			testNames = [];
+			for ( j = 0, tl = config.modules[ i ].tests.length; j < tl; j++ ) {
+				testName = uniqueTestName( testNames,
+					config.modules[ i ].tests[ j ].name );
```

Comment:
Ideally we'd avoid this duplication. But at least worth leaving a comment here to point to the authoratative place in test.js where the same code is duplicated from. And in the other direction as well.

Reply:
@Krinkle Added comments in both places.